File: noi.openjudge.cn/ch0105/24.py
# http://noi.openjudge.cn/ch0105/solution/39684519/

n = int(input())
s = 0
m = 0
for x in range(n):
    a, b = map(int, input().split())
    if 90 <= a <= 140 and 60 <= b <= 90:
        s += 1
    else:
        s = 0
    if s > m:
        m = s
print(m)

File: noi.openjudge.cn/ch0107/24.cpp
// http://noi.openjudge.cn/ch0107/solution/40961367/

#include<iostream>
#include<string>
using namespace std;
string a;
int sum;
int main() {
    getline(cin, a);
    for (int i=0;i<a.size();i++) {
        if (a[i] != ' ') {
            sum++;
        }
        if (sum == 0 && a[i] == ' ') {
            continue;
        }
        if (a[i] == ' ') {
            cout << sum << ',';
            sum = 0;
        }
    }
    cout << sum;
}
